code/obj/item/clothing/uniforms.dm

```diff
@@ -897,18 +897,86 @@
 	icon_state = "referee"
 	item_state = "referee"
 
-/obj/item/clothing/under/shirt_pants
-	name = "shirt and pants"
-	desc = "A button-down shirt and some pants."
+//Black Pants
+/obj/item/clothing/under/shirt_pants_b
+	name = "shirt and black pants"
+	desc = "A button-down shirt and some black pants."
 	icon = 'icons/obj/clothing/uniforms/item_js_misc.dmi'
 	wear_image_icon = 'icons/mob/jumpsuits/worn_js_misc.dmi'
 	icon_state = "shirt_n_pant-b"
 	item_state = "shirt_n_pant-b"
 
-	New()
-		..()
-		src.icon_state = "shirt_n_pant-[pick("b", "br", "w")][pick("", "_tie-r", "_tie-b", "_tie-bl")]"
-		src.item_state = "[src.icon_state]"
```

Comment:
i think making a separate outfit for random generated shirtpants and leaving that as the default for jobs that had it, and leaving all these options for the vendor. gives the jobs a bit of (small) personality

Reply:
Good point

Reply:
Done